tests/e2e/test_rosacli_machine_pool.go

```diff
@@ -660,12 +660,19 @@ var _ = Describe("Create machinepool",
 				Expect(err).ToNot(HaveOccurred())
 
 				By("Create temporary account-roles for instance type list")
-				accountRoles, err := ph.PrepareAccountRoles(rosaClient, "test-55979",
-					false,
-					clusterConfig.Version.RawID,
-					clusterConfig.Version.ChannelGroup,
-					"", "")
+				namePrefix := "test-55979"
```

Comment:
We found some ci failure on the cases using fix prefix or clusterName because some jobs would be executed in parallel. Could you use the namePrefix with some random characters, like https://github.com/openshift/rosa/pull/2421